contents/docs/experiments/common-questions.mdx

```diff
@@ -61,5 +61,5 @@ A situation where this happens is using pageviews as your goal metric. Because p
 
 To fix this, you make sure flags are immediately available on a page load. There are two options to do this:
 
-1. Wait for feature flags to load before showing the page (low engineering effort, but slows page down by ~200ms).
+1. [Wait for feature flags to load](/docs/feature-flags/adding-feature-flag-code#ensuring-flags-are-loaded-before-usage) before showing the page (low engineering effort, but slows page down by ~200ms).
```

Comment:
hey @slshults, don't think that link does everything we need here (missing the part where you're blocking any content showing up on the screen) - but if you feel this is enough of a starting point to make things clear, works 👍

Reply:
Thanks @neilkakkar ! Do we have any existing docs with "the part where you're blocking any content showing up on the screen" mentioned already? (Not exactly sure what you mean, so my searches for it are likely not so accurate.)

Reply:
hmm, I remember it being in a community question, but not in docs. I'll see if I can find it.